### ISSUE TITLE
'Tensor' object has no attribute 'items'

### ISSUE THREAD

**Issue by BOTAK0803** [lsheng23/Practicum#1]
<img width="984" alt="截屏2021-05-20 12 29 54" src="https://user-images.githubusercontent.com/22768966/118919440-1630d080-b967-11eb-9590-1d71c920fefa.png">

**Comment by xavierad** [comment]
I solved it by making `for idx, ff in enumerate(bag_features)`

**Comment by xavierad** [comment]
Indeed I think there will be raised more errors, such as one related to Linear layer. It is due to `X.reshape([1, 7*512])` that is not generalized

**Comment by hansenmaster** [comment]
@xavierad @BOTAKKK Did  you solve the problem? I am stuck on this error too

**Comment by xavierad** [comment]
Yes. Like I explained. 
However I think this approach of building bags of features requires a lot o memory storage. If you want I can put here the link to another repo

**Comment by hansenmaster** [comment]
@xavierad yes please, if you have the solution on more detail please share. Thanks

**Comment by xavierad** [comment]
Here you have
https://github.com/AMLab-Amsterdam/AttentionDeepMIL

**Comment by hansenmaster** [comment]
> Here you have
> https://github.com/AMLab-Amsterdam/AttentionDeepMIL

Thank you very much, it seems legit

**Comment by genhao3** [comment]
> enumerate

and then? Is this problem solved?
